trafficgen-server-start

```diff
@@ -108,18 +108,34 @@ if [ "$switch_type" == "testpmd" ]; then
 
     # Build testpmd cmdline opts for device selection
     echo "Resolving devices for testpmd based on devices [$devices]"
-    testpmd_dev_opt=""
-    for dev in `echo $devices | sed -e 's/,/ /g'`; do
-        res_dev=""
-        resolve_device res_dev $dev
-        testpmd_dev_opt+=" --allow $res_dev"
-        if [ ! -z "$testpmd_devopt" ]; then
-            testpmd_dev_opt+=",$testpmd_devopt"
-        fi
-    done
-    if [ -z "$testpmd_dev_opt" ]; then
+    testpmd_devices=""
+    dev_1=$(echo "${devices}" | cut -d, -f 1)
+    dev_2=$(echo "${devices}" | cut -d, -f 2)
+    dev_3=$(echo "${devices}" | cut -d, -f 3-)
```

Comment:
what is this "-" doing here at the end ?

Reply:
The `-` makes it return all fields from 3 to N.  In theory you could have something like `<foo>,<foo>,,<foo>` that wouldn't be caught if you only used `3` instead of `3-`.